ariadne/src/Ariadne/Wallet/Cardano/Kernel/DB/HdWallet.hs

```diff
@@ -132,23 +139,27 @@ deriveSafeCopySimple 1 'base ''HasSpendingPassword
 -------------------------------------------------------------------------------}
 
 -- | HD wallet root ID
-data HdRootId = HdRootId (InDb (Core.AddressHash Core.PublicKey))
-  deriving (Eq, Ord)
+data HdRootId = HdRootId
```

Comment:
Maybe replace this with a `newtype` and use `GeneralizedNewtypeDeriving` for `Show`?

Reply:
:ok_hand: